lib/beacon_web/plugs/put_root_layout.ex

```diff
@@ -9,6 +9,6 @@ defmodule BeaconWeb.Plug do
 
   @impl Plug
   def call(%Conn{} = conn, _) do
-    Phoenix.Controller.put_root_layout(conn, {BeaconWeb.LayoutView, "root.html"})
+    Phoenix.Controller.put_root_layout(conn, {BeaconWeb.LayoutView, :root})
```

Comment:
This is causing dialyzer errors on the user-project repo. Could be fixed by https://github.com/phoenixframework/phoenix/pull/5167